snuba/utils/streams/topics.py

```diff
@@ -28,6 +28,8 @@ class Topic(Enum):
     QUERYLOG = "snuba-queries"
     PROFILES = "processed-profiles"
     DEAD_LETTER_QUEUE_INSERTS = "snuba-dead-letter-inserts"
+    DEAD_LETTER_METRICS = "snuba-dead-letter-metrics"
+    DEAD_LETTER_SESSIONS = "snuba-dead-letter-sessions"
```

Comment:
is the sessions DLQ addition intended?

Reply:
Yeah, both of these topics exist now so I just added it here to use when we deploy a DLQ on sessions.